src/Simplify.elm

```diff
@@ -7455,6 +7459,12 @@ type alias WrapperProperties otherProperties =
         }
 
 
+type alias FromListProperties otherProperties =
+    { otherProperties
```

Comment:
All other `-Properties` types start by requiring the default `TypeProperties`. Maybe add this here, too, for consistency?
```elm
type alias FromListProperties otherProperties =
    TypeProperties
        { otherProperties | ... }
```